front-page.php

```diff
@@ -175,9 +165,10 @@
 
       <div class="col s8">
 
-        <img src="https://placeholdit.imgix.net/~text?txtsize=50&txt=AD&w=400&h=400">
-
-        <img src="https://placeholdit.imgix.net/~text?txtsize=50&txt=AD&w=400&h=400">
+        <div class="feed-post-container col s6">
```

Comment:
the thing about advert widths and floating. why does the s6 need to be used tho? cant `.feed-post-container` just float?